rotkehlchen/tests/utils/premium.py

```diff
@@ -140,7 +138,7 @@ def setup_starting_environment(
         newer_remote_db: bool,
         db_can_sync_setting: bool,
         premium_credentials: PremiumCredentials,
-        remote_data: bytes = REMOTE_DATA,
+        remote_data: Optional[bytes] = DEFAULT_DATA,
```

Comment:
Change this. No reason to have a weird "magic" constant like this.
```suggestion
        given_remote_data: Optional[bytes] = None,
```

Reply:
I used this instead of `None` because because `None` is a valid value at https://github.com/rotki/rotki/blob/83483160d3d431f98950c5dd8e786098716361de/rotkehlchen/tests/utils/premium.py#L182